pyccel/codegen/printing/ccode.py

```diff
@@ -689,7 +689,7 @@ def _print_NumpyRandint(self, expr):
     def _print_FunctionDef(self, expr):
 
         if len(expr.results) > 1:
-            self._additional_args = expr.results
+            self._additional_args.append(expr.results)
```

Comment:
You need to initialise `self._additional_args` in the `__init__` method

Reply:
it's already set to empty in __init__   line  227

Reply:
Ah, my bad. Sorry